src/index/Index.cpp

```diff
@@ -885,24 +896,19 @@ void Index::createFromOnDiskIndex(const string& onDiskBase,
   setOnDiskBase(onDiskBase);
   _vocab.readFromFile(_onDiskBase + ".vocabulary",
                       _onDiskLiterals ? _onDiskBase + ".literals-index" : "");
+  // this is the read only version, TODO: give better naming or type
+  _opsMeta.setup(onDiskBase + MMAP_FILE_SUFFIX + ".ops");
```

Comment:
Isn't `setup()` basically `open()` so could be named `open()`?

Reply:
Well, open() would suggest that we open a file, but actually what we are doing is completely depending on our implementation (nothing, reloading a mmapBased array, copying something to a stxxl vector (deprecated) etc.
My purpose was to create a generic function where the different template instatiations can have different initializations/setups passed through by varargs. I chose "setup" because you anyway have to look at the concrete type and which type of setup is required.

Reply:
Sounds reasonable